openshift/ansible/__init__.py

```diff
@@ -35,15 +40,25 @@ def __str__(self):
 
 
 class OpenShiftAnsibleModule(AnsibleModule):
+    """
+    Abstraction of an OpenShift Ansible Module.
+    """
+
     def __init__(self, openshift_type):
+        """
```

Comment:
[Here is an example generator](https://gist.github.com/ashcrow/83c9ba0ad7cedfe3345a79600838369e) I whipped up just now that you may be able to use depending on your editor.